contentcuration/contentcuration/settings.py

```diff
@@ -86,6 +87,16 @@
     }
 }
 
+# READ-ONLY SETTINGS
+# Set STUDIO_INCIDENT_TYPE to a key from contentcuration.utils.incidents to activate
+INCIDENT = INCIDENTS.get(os.getenv('STUDIO_INCIDENT_TYPE'))
+SITE_READ_ONLY = INCIDENT and INCIDENT['readonly']
```

Comment:
This value derived might be more readable if it's set as `SITE_READ_ONLY = INCIDENT.get("readonly")`. But this is a minor nitpick.

Reply:
If INCIDENT isn't set, it will throw an AttributeError though